b2b-sample/README.md

```diff
@@ -299,7 +299,7 @@ setup the config json file as mentioned above.
 - Open [http://localhost:3001](http://localhost:3001) with your browser to see the result.
 - Click `Sign in`.
 - Select the `Sign in With SSO` option.
-- Type `Best Car Mart` when pop up to type the organization name and click `Submit`.
+- Type `bestcarmart` when pop up to type the organization handle and click `Submit`.
```

Comment:
Improve "### Step 1: Create an organization" guide to enter 
Best Car Mart as organization name and bestcarmart as organization handle